src/equations/shallow_water_multilayer_1d.jl

```diff
@@ -353,6 +376,79 @@ In the two-layer setting this combination is equivalent to the fluxes in:
     return SVector(f)
 end
 
+"""
+    hydrostatic_reconstruction_ersing_etal(u_ll, u_rr, equations::ShallowWaterMultiLayerEquations1D)
+
+!!! warning "Experimental code"
+    This is an experimental feature and may change in future releases.
+
+A particular type of hydrostatic reconstruction of the water height and bottom topography to 
+guarantee well-balancedness in the presence of wet/dry transitions and entropy stability for the 
+[`ShallowWaterMultiLayerEquations1D`](@ref). 
+The reconstructed solution states `u_ll_star` and `u_rr_star` variables are used to evaluate the 
+surface numerical flux at the interface. The key idea is a piecewise linear reconstruction of the 
+bottom topography and water height interfaces using subcells, where the bottom topography is allowed 
+to be discontinuous. 
+Use in combination with the generic numerical flux routine [`Trixi.FluxHydrostaticReconstruction`](@extref).
+"""
```

Comment:
```suggestion
"""
    hydrostatic_reconstruction_ersing_etal(u_ll, u_rr, equations::ShallowWaterMultiLayerEquations1D)

A particular type of hydrostatic reconstruction of the water height and bottom topography to 
guarantee well-balancedness in the presence of wet/dry transitions and entropy stability for the 
[`ShallowWaterMultiLayerEquations1D`](@ref). 
The reconstructed solution states `u_ll_star` and `u_rr_star` variables are used to evaluate the 
surface numerical flux at the interface. The key idea is a piecewise linear reconstruction of the 
bottom topography and water height interfaces using subcells, where the bottom topography is allowed 
to be discontinuous. 
Use in combination with the generic numerical flux routine [`Trixi.FluxHydrostaticReconstruction`](@extref).

!!! warning "Experimental code"
    This is an experimental feature and may change in future releases.
"""
```
I prefer to stick the experimental warning to be last part of the doc string.

Reply:
Sure, I changed this in 3d6d89a9381cd544f4ee470550c38ee47df42bf1